acceptance-setup/__tests__/acceptance.js

```diff
@@ -0,0 +1,175 @@
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+
+const server = require('../server');
+const { cli, Registry } = require('oc');
+const path = require('path');
+const r = require('request-promise-native');
+const jsdom = require('jsdom');
+const JSDOM = jsdom.JSDOM;
+const fs = require('fs-extra');
+
+jest.unmock('minimal-request');
```

Comment:
not a big fan of this here at the beginning...shouldn't it be on an after() step inside some unit instead?